native_client/deepspeech.cc

```diff
@@ -49,6 +61,11 @@ Model::Model(const char* aModelPath, int aNCep, int aNContext,
   mPriv->ncontext = aNContext;
 
   mPriv->alphabet = new Alphabet(aAlphabetConfigPath);
+
+  mPriv->scorer = new KenLMBeamScorer(aLMPath, aTriePath, aAlphabetConfigPath,
```

Comment:
I think there would be value, especially for experimenting performances (memory, inference time, word error rate) on some platform, to be able to bypass the use of the language model.